homeassistant/components/aemet/weather_update_coordinator.py

```diff
@@ -286,7 +286,7 @@ def _convert_weather_response(self, weather_response):
         temperature_feeling = None
         town_id = None
         town_name = None
-        town_timestamp = dt_util.as_utc(elaborated).isoformat()
+        town_timestamp = dt_util.as_utc(elaborated)
```

Comment:
If nothing else depends on this and `elaborated` is already timezone aware, we could just return that without converting to UTC time. The base sensor class will convert to UTC time for us.

Reply:
I think it's better to convert to force the UTC conversion to keep consistency with the rest of the code (station_timestamp, daily/hourly ATTR_FORECAST_TIME).

Reply:
Ok.